device/bgp/bgproutepathattributes.yaml

```diff
@@ -18,4 +18,10 @@ components:
           enum:
           - igp
           - egp
-          default: igp
\ No newline at end of file
+          default: igp
+        local_preference:
+          description: >-
+            Local Preference attribute, for iBGP peers to select the path for the traffic leaving the AS.
+          type: integer
+          default: 100
```

Comment:
Maybe default of 0 is better since it aligns with current defaults and with default  in IxN today.

Reply:
what's wrong with 100 as a default value which is more universally recognized ?